python/lsst/sitcom/vandv/m1m3.py

```diff
@@ -30,14 +30,14 @@ def get_rms(s):
     return rms
 
 
-def lut_elevation_forces(elevation, lut_fname, lut_path=None):
+def lut_elevation_forces(elevation, lut_fname, lut_path=None, as_array=False):
```

Comment:
Could you add the documentation for `as_array` in this function and in the functions below?